### PR TITLE
Add recipes for multi_key_dict, mailer, python-jenkins and jobs_done10 [skip appveyor]

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
All jobs passed 🎉 

@conda-forge/staged-recipes @igortg @ocefpaf I believe this can now be reviewed. 👍

File: recipes/python-jenkins/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+requirements:
+  host:
+    - pbr >=0.8.2
+    - pip
+    - python
+  run:
+    - multi_key_dict
+    - python
+    - requests
+    - setuptools  # requires pkg_resources at runtime
```
Comment (marcelotrevisani) [inline recipes/python-jenkins/meta.yaml]: Oh, thanks for the comment! hahah
:+1:

**Comment by marcelotrevisani** [comment]
Looks good to me! I'm going to merge this!
Thanks!  :)

Ps: I will start to read your book today, hahah

File: recipes/mailer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 3411a12197731e0d5379ab194d9acc8d0d71c8b95fdfb11474d67c3e9860070e
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/mailer/meta.yaml]: Just one thing here, is it possible to this recipe be ``noarch: python`` ?

**Comment by marcelotrevisani** [comment]
I just saw one thing to be improve, I think the recipe ``mailer`` may be ``noarch: python``

File: recipes/mailer/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/mailer/meta.yaml]: It looks like a python project for me

**Comment by marcelotrevisani** [comment]
After my considerations I think that should be good to go! :)

File: recipes/mailer/meta.yaml (hunk 2 above)
Comment (nicoddemus) [reply recipes/mailer/meta.yaml]: No, unfortunately it uses 2to3 during installation time

Comment (marcelotrevisani) [reply recipes/mailer/meta.yaml]: Ah, ok then! I'm going to merge this!

Thanks to double check that  :)

File: recipes/python-jenkins/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - six >=1.3.0
+
+test:
+  imports:
+    - jenkins
+
+about:
+  home: http://git.openstack.org/cgit/openstack/python-jenkins
+  license: BSD
```
Comment (ocefpaf) [inline recipes/python-jenkins/meta.yaml]: Version?

Comment (ocefpaf) [reply recipes/python-jenkins/meta.yaml]: Looks like it is BSD-3-Clause: http://git.openstack.org/cgit/openstack/python-jenkins/tree/COPYING

**Pull request merged**